modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'vba_documents/upload_error'
+
+# TODO: add support for base64 encoding
+module VBADocuments
```

Comment:
I might have missed this discussion, but did you consider Rack::Multipart::Parser?

Reply:
I looked briefly...since we don't have access to the Content-Type header with boundary we at least needed to extract that out, though I guess we could then rewind and use R::M::P. Willing to consider swapping that in.